### PR TITLE
fix(UI): don't run shutter animation twice

### PR DESCRIPTION
**Pull request by andi34**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/PhotoboothProject/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [ ] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)
Cheese image depends on activated shutter animation.
On Photobooth v4.0.0 the shutter animation was started if an image was defined independent of the shutter animation. api.shutter.stop is only executed inside api.callTakePicApi if shutter Animation is enabled.
There might be cases where api.shutter.start was executed twice, make sure it's only called once.
Start api.shutter.start where needed depending on our settings.

### CODE REVIEW

**Comment by up-87** [comment]
Ah see... I didn't expect this to be set when the animation was not enabled. But yeah... It can happen especially if so plays around with the config. 👍

File: src/js/core.js [hunk 1]
```diff
@@ -457,7 +457,7 @@ const photoBooth = (function () {
 
     api.cheese = function (photoStyle) {
         cheese.empty();
-        if (config.ui.shutter_cheese_img !== '') {
+        if (config.ui.shutter_animation && config.ui.shutter_cheese_img !== '') {
```
Comment (andi34) [inline src/js/core.js:460]: Start shutter here if cheese image is set

File: src/js/core.js [hunk 2]
```diff
@@ -496,7 +496,7 @@ const photoBooth = (function () {
     };
 
     api.callTakePicApi = function (data, retry = 0) {
-        if (config.ui.shutter_animation) {
+        if (config.ui.shutter_animation && config.ui.shutter_cheese_img === '') {
```
Comment (andi34) [inline src/js/core.js:499]: Start shutter here if shutter image is not set

**Comment by andi34** [comment]
@up-87 I've also not expected it.

Should be fine this way, but haven't tested yet.
Will try to test this weekend, if you can test before and confirm everything works you can merge already.

**Pull request merged**